code/modules/mob/living/carbon/human/species_types/rage.dm

```diff
@@ -0,0 +1,56 @@
+/*Result of the rage serum, turning players into what is essentially a walking tank that exists solely to smash those weaker than them. Extremely durable
+but essentially loses their ability to interact in any meaningful way that doesn't involve slamming their fists against it. */
+
+/datum/species/rage
+	name = "Walking Mass of Muscle and Testosterone."
+	id = "buff_hulk"
+	say_mod = "grunts"
+	blacklisted = TRUE
+	default_color = "34AF10"
+	fixed_mut_color = "34AF10"
+	species_traits = list(MUTCOLORS,EYECOLOR,HAIR,FACEHAIR,LIPS,PIERCEIMMUNE,NODISMEMBER,RADIMMUNE)
+	use_skintones = 0
+	mutant_bodyparts = list("tail_human", "ears", "wings")
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	siemens_coeff = 0
+	limbs_id = "human"
+	no_equip = list(slot_head, slot_gloves, slot_wear_suit) //keeps user from wearing most types of armor to prevent additional resistance stacking.
+	armor = 45
+	speedmod = 0
+	brutemod = 0.25
+	burnmod = 0.35
+	heatmod = 1.2
+	stunmod = 0.15
```

Comment:
Some things about this:
- Armor is percent reduction in damage, and stacks with the rest. This means that brute damage is actually ~13% and so on.
- Heatmod determines the damage from heat, but it's also multiplied by the burnmod AND the armor. Heat is not a weakness here.
- Speedmod 0 is base speed, you don't need to specify it.